docs/resources/Channel.md

```diff
@@ -884,39 +884,39 @@ Post a message to a guild text or DM channel. Returns a [message](#DOCS_RESOURCE
 You may create a message as a reply to another message. To do so, include a [`message_reference`](#DOCS_RESOURCES_CHANNEL/message-reference-object-message-reference-structure) with a `message_id`. The `channel_id` and `guild_id` in the `message_reference` are optional, but will be validated if provided.
 
 > info
-> Note that when sending a message, you must provide a value for at **least one of** `content`, `embed`, or `file`.
+> Note that when sending a message, you must provide a value for at **least one of** `content`, `embeds`, or `file`.
 
 > info
 > For a `file` attachment, the `Content-Disposition` subpart header MUST contain a `filename` parameter.
 
 > warn
 > This endpoint supports both `application/json` and `multipart/form-data` bodies. When uploading files the `multipart/form-data` content type must be used.
-> Note that in multipart form data, the `embed` and `allowed_mentions` fields cannot be used. You can pass a stringified JSON body as a form value as `payload_json` instead.
+> Note that in multipart form data, the `embeds` and `allowed_mentions` fields cannot be used. You can pass a stringified JSON body as a form value as `payload_json` instead.
 > **If you supply a `payload_json` form value, all fields except for `file` fields will be ignored in the form data**.
 
 ###### JSON/Form Params
 
-| Field             | Type                                                                                              | Description                                                  | Required                    |
-| ----------------- | ------------------------------------------------------------------------------------------------- | ------------------------------------------------------------ | --------------------------- |
-| content           | string                                                                                            | the message contents (up to 2000 characters)                 | one of content, file, embed |
-| tts               | boolean                                                                                           | true if this is a TTS message                                | false                       |
-| file              | file contents                                                                                     | the contents of the file being sent                          | one of content, file, embed |
-| embed             | [embed](#DOCS_RESOURCES_CHANNEL/embed-object) object                                              | embedded `rich` content                                      | one of content, file, embed |
```

Comment:
Rather than deleting this I would note that it's deprecated and to use the array instead. It's still on the data model so it could still be received.

Reply:
only sending changed; receiving and the documented message object has always had an `embeds` array so there's no change to that